### PR TITLE
Add more explicit instructions to db:tunnel

### PR DESCRIPTION
**Pull request by wcpines**
Should eliminate any confusion about how this is performed.

### CODE REVIEW

**Comment by wcpines** [comment]
@fancyremarker for review. 

Also,  do you think it's worth adding a section on how to use db:tunnel for non command-line db clients?

**Comment by gib** [comment]
Thanks @wcpines!
Does `a separate terminal window` need to be emboldened? The examples great! Did you reference https://www.aptible.com/blog/most_popular_databases.html?

**Comment by wcpines** [comment]
@gib, thank you 

My $0.02: I don't think we need to bold the font since it has its own line. 

Examples were from experience ;)

**Comment by fancyremarker** [comment]
This is great. If you want to make it easier to understand `db:tunnel` for non-CLI clients, we can do one of two things:
1. Explain the components of a URL in this support article, or
2. (Arguably better) Print the components in the `aptible db:tunnel` output, i.e.:

```
Connect at postgresql://aptible:foobar@127.0.0.1:49153/db.
Or, use the following arguments:
  Host: 127.0.0.1 
  Port: 49153
  Username: aptible
  Password: foobar
  Database: db
```

**Comment by wcpines** [comment]
I don't love that this mixes dummy values (`foobar`)  with values that will always be used (`db`).  This could confuse a first-timer.

**Comment by skylar-anderson** [comment]
@fancyremarker -- I think your 2nd suggestion above is terrific.

**Comment by wcpines** [comment]
How's this for compromise?

**Comment by skylar-anderson** [comment]
@wcpines -- Frank is suggesting that we not update the article, but rather update the output of the command itself.  The `foobar` in his example would be the actual DB's password.

**Comment by wcpines** [comment]
Ahhhh, _print_ as in to the shell session, not the article.  

I dig!

**Comment by fancyremarker** [comment]
@wcpines: I implemented what we discussed above in aptible/aptible-cli#123. I've released that in aptible-cli v0.5.12, so you can run `gem install -N aptible-cli` to download it, see how it works, and update the documentation here as appropriate.

**Comment by wcpines** [comment]
@fancyremarker for review of latest addition.

File: source/topics/cli/how-to-connect-to-database-from-outside.md [hunk 1]
```diff
@@ -6,4 +6,14 @@ To connect to your Aptible database from outside Aptible's private cloud, use th
 
 This will create a secure (SSH) tunnel to your database and print a local URL where you can connect to the database.
 
-Open a separate terminal window and pass in the URL using the appropriate database client. 
+Open a separate terminal window and, depending on the database type:
+
+- Pass in the entire URL, e.g. 
+
+      psql postgresql://aptible:[PASSWORD]@127.0.0.1:[PORT]/db
```
Comment (fancyremarker) [inline source/topics/cli/how-to-connect-to-database-from-outside.md]: This needs 2 more (8 total) spaces to be properly syntax highlighted.

**Comment by wcpines** [comment]
That's concerning. Marked2 shows proper highlight: 

image

**Comment by gib** [comment]
:+1: :+1:

**Comment by chasballew** [comment]
@wcpines - You will want to use a Markdown parser that uses GitHub-flavored Markdown. Code blocks in a list need to be double-indented.

**Comment by wcpines** [comment]
@chasballew Marked2 has a github option (you can see it selected in the screencap above) I'll consider other options since this one isn't reliable.  Maybe a bug ticket to them too.

**Comment by chasballew** [comment]
You can also use the GitHub web UI to confirm the view.

**Pull request merged**